rest_framework/templates/rest_framework/base.html

```diff
@@ -286,7 +286,7 @@ <h1>{{ name }}</h1>
       <script>
         window.drf = {
           csrfHeaderName: "{{ csrf_header_name|default:'X-CSRFToken' }}",
-          csrfCookieName: "{{ csrf_cookie_name|default:'csrftoken' }}"
+          csrfToken: "{% if request %}{{ csrf_token }}{% endif %}"
```

Comment:
Do you mean `{% csrf_token %}`? c.f. #3703 &co.

Reply:
How very odd... I am *sure* I used it this way when I was first figuring things out, clearly I was mistaken. OK, I'll have to re-work one test too, will take care of that in the next day or two.

Reply:
Oh, no, I do want `{{ csrf_token }}`. I don't want the entire HTML tag, just the value of the token.

Reply:
Good, and `TemplateHTMLRenderer` will create a `RequestContext`, so `csrf_token` will be in the context. Fine. 👍